Sorting Algorithms/wavesort.cpp

```diff
@@ -0,0 +1,73 @@
+Given a sorted array arr[] of distinct integers. Sort the array into a wave-like array and return it
```

Comment:
Write the problem statement as a multi-line comment.

Reply:
Okay

Reply:
i have added the multi line comment